### ISSUE TITLE
End-to-end Deep Learning Methods for Automated Damage Detection in Extreme Events at Various Scales. (arXiv:2011.03098v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#580]
End-to-end Deep Learning Methods for Automated Damage Detection in Extreme Events at Various Scales. (arXiv:2011.03098v1 [cs.CV])<br>
https://ift.tt/3eFGHYJ<br>
<p>Robust Mask R-CNN (Mask Regional Convolu-tional Neural Network) methods are proposed and tested for automatic detection of cracks on structures or their components that may be damaged during extreme events, such as earth-quakes. We curated a new dataset with 2,021 labeled images for training and validation and aimed to find end-to-end deep neural networks for crack detection in the field. With data augmentation and parameters fine-tuning, Path Aggregation Network (PANet) with spatial attention mechanisms and High-resolution Network (HRNet) are introduced into Mask R-CNNs. The tests on three public datasets with low- or high-resolution images demonstrate that the proposed methods can achieve a big improvement over alternative networks, so the proposed method may be sufficient for crack detection for a variety of scales in real applications.</p>
<br>
<br>
via cs.CV updates on arXiv.org http://arxiv.org/<br>
